recipes-kernel/linux/files/set-cmdline.cfg

```diff
@@ -0,0 +1 @@
+CONFIG_CMDLINE="rw console=hvc0 clk_ignore_unused earlycon=xenboot debug root=/dev/ram0"
```

Comment:
where this will go? cmdline.txt at RPI5 bootfs?

Reply:
No. Linux .config

Reply:
have you checked if cmdline.txt override cmdline on Linux dtb or not? That's important i think